client/src/components/SocialLoginBtn/socialLoginBtn.scss

```
@@ -0,0 +1,18 @@
+.login-btn-wrap {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-bottom: 1em;
+  background-color: lightgray;
+  border-radius: 0.7em;
+  cursor: pointer;
+
+  .login-text {
+    margin-left: auto;
+    margin-right: auto;
+  }
```

margin: 0 auto; 를 사용하시면 한줄로 변경 가능합니다!

```suggestion
  .login-text {
    margin: 0 auto;
  }
```